CHANGELOG.md

```diff
@@ -3,6 +3,50 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/) and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+## Next - Unreleased
```

Comment:
De CHANGELOG.md change is te groot: Het lijkt erop alsof je vorige entries ook toevoegd. Daarbij heeft dit PR een merge conflict, als je het merge conflict resolved door master in deze branch te mergen zal het er ook al heel anders uitzien

Reply:
Na jouw laatste change is de CHANGELOG.md file uiteindelijk zo geworden: https://github.com/dso-toolkit/dso-toolkit/blob/ac11a2fae16018e64ccc9266a6dc6bb32763de0e/CHANGELOG.md

Dit is ook niet de bedoeling

Reply:
In alle eerlijkheid heb ik geen flauw idee hoe ik deze nu moet rechtbrijen. Want als ik nu de changelog weer aanpas zul je al die veranderingen weer zien. Of ja alle zinnetjes die ik weer heb toegevoegd.  Of niet?

Reply:
Ik heb de changelog toegevoegd. Maar daar was ik al bang voor heel de changelog is nu een groot groen blok.